frontend/src/views/File.vue

```diff
@@ -80,6 +101,32 @@ main#filelanding
           </span>
       </div>
       <div class="actions">
+        <a class="secondaryButton"
```

Comment:
Voisi näyttää paremmalta (volatiili tiedostojen kanssa) jos ensin on "How to cite" ja sitten "License". Napit menisi silloin vaaleasta tummempaan päin.